templates/single/assignment/single-attempt.php

```diff
@@ -0,0 +1,161 @@
+	$pass_mark  = tutor_utils()->get_assignment_option( $submitted_assignment->comment_post_ID, 'pass_mark' );
+	$given_mark = get_comment_meta( $submitted_assignment->comment_ID, 'assignment_mark', true );
+?>
+<div class="tutor-assignment-result-table tutor-mt-32 tutor-mb-40">
+	<div class="tutor-table-responsive">
+	<table class="tutor-table my-quiz-attempts">
+		<thead>
+		<tr>
+			<th><?php esc_html_e( 'Date', 'tutor' ); ?></th>
+			<th><?php esc_html_e( 'Total Marks', 'tutor' ); ?></th>
+			<th><?php esc_html_e( 'Pass Marks', 'tutor' ); ?></th>
+			<th><?php esc_html_e( 'Earned Marks', 'tutor' ); ?></th>
+			<th><?php esc_html_e( 'Result', 'tutor' ); ?></th>
+		</tr>
+		</thead>
+
+		<tbody>
+		<tr>
+			<td>
+				<?php echo esc_html( DateTimeHelper::get_gmt_to_user_timezone_date( $submitted_assignment->comment_date_gmt ) ); ?>
+			</td>
+			<td><?php esc_html_e( $total_mark, 'tutor' );//phpcs:ignore ?></td>
+			<td><?php esc_html_e( $pass_mark, 'tutor' );//phpcs:ignore ?></td>
+			<td><?php esc_html_e( $given_mark, 'tutor' );//phpcs:ignore ?></td>
+			<td>
+				<?php if ( $is_reviewed_by_instructor ) : ?>
+					<?php if ( $given_mark >= $pass_mark ) : ?>
+						<span class="tutor-badge-label label-success">
+							<?php esc_html_e( 'Passed', 'tutor' ); ?>
+						</span>
+					<?php else : ?>
+						<span class="tutor-badge-label label-danger">
+							<?php esc_html_e( 'Failed', 'tutor' ); ?>
+						</span>
+					<?php endif; ?>
+				<?php endif; ?>
+		
+				<?php if ( ! $is_reviewed_by_instructor ) : ?>
+					<span class="tutor-badge-label label-warning">
+						<?php esc_html_e( 'Pending', 'tutor' ); ?>
+					</span>
+				<?php endif; ?>
+			</td>
+		</tr>
+		</tbody>
+	</table>
+	</div>
+</div>
+
+<?php
+	$instructor_note = get_comment_meta( $submitted_assignment->comment_ID, 'instructor_note', true );
+if ( ! empty( $instructor_note ) && $is_reviewed_by_instructor ) :
+	?>
+	<div class="tutor-instructor-note tutor-my-32 tutor-py-20 tutor-px-24 tutor-py-sm-32 tutor-px-sm-36">
+	<div class="tutor-in-title tutor-fs-6 tutor-fw-medium tutor-color-black">
+	<?php esc_html_e( 'Instructor Note', 'tutor' ); ?>
+	</div>
+	<div class="tutor-in-body tutor-fs-6 tutor-color-secondary tutor-pt-12 tutor-pt-sm-16">
+	<?php echo wp_kses_post( nl2br( get_comment_meta( $submitted_assignment->comment_ID, 'instructor_note', true ) ) ); ?>
+	</div>
+</div>
+<?php endif; ?>
+
+<div class="tutor-assignment-details  tutor-pb-48 tutor-pb-sm-72">
+	<div class="tutor-ar-body tutor-pt-24 tutor-pb-40 tutor-px-16 tutor-px-md-32">
+		<div class="tutor-ar-header tutor-d-flex tutor-justify-between tutor-align-center">
+			<div class="tutor-ar-title tutor-fs-6 tutor-fw-medium tutor-color-black">
+				<?php esc_html_e( 'Your Assignment', 'tutor' ); ?>
+			</div>
+
+			<?php
+			$result = tutor()->has_pro ? \TUTOR_ASSIGNMENTS\Assignments::get_assignment_result( $post_id, $user_id ) : 'pending';
```

Comment:
If there is no PRO, then the template should not load. The entire assignment template should be in the assignment addon.